include/aoclient.h

```diff
@@ -329,7 +330,8 @@ class AOClient : public QObject {
         {"unban", {ACLFlags.value("BAN"), 1, &AOClient::cmdUnBan}},
         {"removeuser", {ACLFlags.value("MODIFY_USERS"), 1, &AOClient::cmdRemoveUser}},
         {"subtheme", {ACLFlags.value("CM"), 1, &AOClient::cmdSubTheme}},
-        {"about", {ACLFlags.value("NONE"), 0, &AOClient::cmdAbout}}
+        {"about", {ACLFlags.value("NONE"), 0, &AOClient::cmdAbout}},
+        {"evidence_swap", {ACLFlags.value("CM"), 2, &AOClient::cmdEvidence_Swap}}
```

Comment:
```suggestion
        {"evidence_swap", {ACLFlags.value("CM"), 2, &AOClient::cmdEvidence_Swap}},
```
doing this will prevent most merge conflicts